packages/face-detection/src/definitions.ts

```diff
@@ -12,18 +12,29 @@ export interface FaceDetectionPlugin {
 /**
  * @since 5.1.0
  */
-export interface ProcessImageOptions {
+export type ProcessImageOptions = (
+  | {
+      /**
+       * The local path to the image file.
+       *
+       * @since 5.1.0
+       */
+      path: string;
+    }
+  | {
+      /**
+       * Represents an image object (Base64 encoded).
+       *
+       * @since 5.1.0
+       */
+      image: string;
+    }
+) & {
   /**
-   * The local path to the image file.
-   *
-   * @since 5.1.0
+   * The options for the face detector.
    */
-  path: string;
-  /**
-   * The classification mode for face detection.
-   */
-  classificationMode: ClassificationMode;
-}
+  options?: FaceDetectorOptions;
```

Comment:
Please remove the `FaceDetectorOptions` interface and just add the properties to `ProcessImageOptions` for now.